quickwit/quickwit-aws/src/error.rs

```diff
@@ -36,6 +36,11 @@ use aws_sdk_s3::operation::get_object::GetObjectError;
 use aws_sdk_s3::operation::head_object::HeadObjectError;
 use aws_sdk_s3::operation::put_object::PutObjectError;
 use aws_sdk_s3::operation::upload_part::UploadPartError;
+#[cfg(feature = "sqs")]
+use aws_sdk_sqs::operation::{
+    change_message_visibility::ChangeMessageVisibilityError,
```

Comment:
nitpick I think you can group all of the sqs conditional complication like this.

```rust
#[cfg(feature = "sqs")]
{
    #[cfg(feature = "sqs")]
use aws_sdk_sqs::operation::{
    change_message_visibility::ChangeMessageVisibilityError,
    delete_message_batch::DeleteMessageBatchError, receive_message::ReceiveMessageError,
};

impl AwsRetryable for ReceiveMessageError {
    fn is_retryable(&self) -> bool {
        false
    }
}

#[cfg(feature = "sqs")]
impl AwsRetryable for DeleteMessageBatchError {
    fn is_retryable(&self) -> bool {
        false
    }
}

#[cfg(feature = "sqs")]
impl AwsRetryable for ChangeMessageVisibilityError {
    fn is_retryable(&self) -> bool {
        false
    }
}
}
```

If you had exports, it was still possible by introducing a module.

```
mod sqs {
 //..
}
pub use sqs::*;

Reply:
The approach with the plain `{}` block doesn´t seem to work. It was added for blocks within function [here](https://github.com/rust-lang/rfcs/pull/16/files) but not for top level blocks. 

Using `mod` is a good idea here, also applying this to the Kinesis part